src/main/java/uk/gov/hmcts/cmc/claimstore/services/notifications/NotificationToDefendantService.java

```diff
@@ -47,6 +48,18 @@ public void notifyDefendant(Claim claim) {
         );
     }
 
+    public void notifyDefendantWhenInterlocutoryJudgementRequested(Claim claim) {
+        Map<String, String> parameters = aggregateParams(claim);
+        parameters.put(CLAIMANT_NAME, claim.getClaimData().getClaimant().getName());
```

Comment:
should we do any checks here? on `getClaimData().getClaimant().getName()` - maybe these are done higher up

Reply:
If any of those were null the data would have been horribly corrupted. The only real complication will come from multi-party when `getClaimant()` will throw an exception because there isn't only one.